TombEngine/Game/animation.h

```diff
@@ -22,6 +22,16 @@ enum class AnimCommandType
 	Flipeffect
 };
 
+struct SparkOffsets
+{
+	int x;
```

Comment:
1) Don't use `int` for types with three components, such as position or velocity. We have special `Vector3i` type for that.
2) `SparkOffsets`, should it ever exist (which I am not sure), should not be placed in `animation.h`.

Reply:
I replaced it into effects.h

Reply:
and I changed it to Vector3i